cmd/config/main.go

```diff
@@ -53,7 +55,10 @@ func New(ctx *ConfigCommandContext) *cobra.Command {
 
 func validateKey(key string) error {
 	validKeys := make(map[string]bool)
-	validKeys["token"] = true
+
+	for _, key := range ConfigAvailableKeys {
+		validKeys[key] = true
+	}
```

Comment:
In todays build we missed the possibility for config set offline!
This change suppose to couple between adding new tokens and make them valid!